bin/commands/stacks/list.js

```diff
@@ -0,0 +1,19 @@
+const { getConfig } = require('../../../lib/services/configuration');
+
+let output = '';
```

Comment:
Intending to use `chalk` for these and get some nice, colored output.  Output strategy in general is TBD.  Spaghetti for the short-term.